### PR TITLE
ul: add page

### PR DESCRIPTION
**Pull request by johncurcio**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by andrik** [comment]
Hi @johncurcio 

Thank you for your submission. Looks like linter founded some issues, would you please fix and push again?

File: pages/linux/ul.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ul
+
+> Performs the underlining of a text.
+> To underline a text is necessary to underline each character separately.
```
Comment (sbrl) [inline pages/linux/ul.md]: What precisely do you mean by this?

Comment (johncurcio) [reply pages/linux/ul.md]: If you want to underline the character H you can write something like ``_\bH`` in a file or using ``echo`` and call ``ul``. However, if you want to underline Hello, you can surround the whole word with underline ``_\bHello_\b``, you need to do this for each character ``_\bH_\be_\bl_\bl_\bo``.

Comment (sbrl) [reply pages/linux/ul.md]: Ah, I see. Perhaps we could reword this to make it clearer. What about this?
```suggestion
> Each character in a given string must be underlined separately.
```
